relays/substrate/src/main.rs

```diff
@@ -524,8 +555,30 @@ async fn estimate_message_delivery_and_dispatch_fee<Fee: Decode, C: Chain, P: En
 	Ok(decoded_response)
 }
 
+fn compute_maximal_size_of_remark(maximal_source_extrinsic_size: u32, maximal_target_extrinsic_size: u32) -> u32 {
```

Comment:
This could be a helper CLI method.

Reply:
TBH this subcommand seems super-awkward to me :) But here it is `./bin/substrate-relay print-message-limits millau-to-rialto`

Reply:
Yeah, you're right indeed it feels weird. Perhaps instead we could do `--remark-size=max-allowed`?